src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -3,19 +3,42 @@
 
 
 class EventTimestamp(QueryFilterTimestampField):
+    """Class that filters events based on the timestamp of the event that occurred."""
+
     _term = u"eventTimestamp"
 
 
 class EventType(FileEventFilterStringField):
+    """Class that filters file events based on event type.
+
+    Available event types are provided as class attributes:
+
+        - :attr:`EventType.CREATED`
+        - :attr:`EventType.DELETED`
+        - :attr:`EventType.EMAILED`
+        - :attr:`EventType.MODIFIED`
+        - :attr:`EventType.READ_BY_APP`
+
+    Example::
+
+        filter = EventType.isin([EventType.READ_BY_APP, EventType.EMAILED])
+
+    """
+
     _term = u"eventType"
 
     CREATED = u"CREATED"
     MODIFIED = u"MODIFIED"
     DELETED = u"DELETED"
     READ_BY_APP = u"READ_BY_APP"
+    EMAILED = u"EMAILED"
 
 
 class InsertionTimestamp(QueryFilterTimestampField):
+    """Class that filters events based on the timestamp of when the event was actually added to the
```

Comment:
For timestamp related fields, I had a thought, let me know what you think.....

link to the `basics` user guide section about dates

?

Reply:
How do I link to the internal .md docs from within a docstring?

Reply:
Oh I don't know - it might have to be a hardcoded link... let's not worry about it then

Reply:
I actually figured it out (after much trial and error):

You can define arbitrary anchors in docs using the `.. _my_anchor:` rst syntax. Which when in a .md file you need to wrap in a eval_rst codeblock:

```
```eval_rst
.. _my_anchor:
```
```
Then you then can link directly to that anchor from anywhere else with: `:ref:\`Link Title <my_anchor>\``

Reply:
Sorry for the edit spam, I just realized there's a Preview option for the comment box 🤦‍♂